adobe_vipm/flows/fulfillment/reseller_transfer.py

```
@@ -181,3 +182,35 @@ def __call__(self, mpt_client, context, next_step):
+            if subscription.get("autoRenewal", {}).get("enabled", True):
+                continue
+
+            subscription_id = subscription["subscriptionId"]
+            try:
+                adobe_client.update_subscription(
+                    context.authorization_id,
+                    context.customer_id,
+                    subscription_id,
+                    auto_renewal=True,
+                )
+            except AdobeAPIError as ex:
```

Ruff (0.14.7)

Local variable `ex` is assigned to but never used

Remove assignment to unused variable `ex`

(F841)